modules/javafx.graphics/src/main/java/com/sun/javafx/sg/prism/NGShape3D.java

```diff
@@ -113,15 +113,16 @@ private void renderMeshView(Graphics g) {
             // is a single white point light at camera eye position.
             meshView.setAmbientLight(0.0f, 0.0f, 0.0f);
             Vec3d cameraPos = g.getCameraNoClone().getPositionInWorld(null);
-            meshView.setPointLight(lightIndex++,
+            meshView.setLight(lightIndex++,
                     (float) cameraPos.x,
                     (float) cameraPos.y,
                     (float) cameraPos.z,
                     1.0f, 1.0f, 1.0f, 1.0f,
                     NGPointLight.getDefaultCa(),
                     NGPointLight.getDefaultLa(),
                     NGPointLight.getDefaultQa(),
-                    NGPointLight.getDefaultMaxRange());
+                    NGPointLight.getDefaultMaxRange(),
+                    0, 0, 1, 0, 180, 0); // simulating point light);
```

Comment:
I had these constants already in `NGPointLight`, but they were accessed in a non-static way so the `setLight` method could be unified for `NGPointLight`s. Since there is no instance of a light in this case (we are artificially making a light), we would need to access them also statically. I made this change, but I'm not sure if it's designed properly.